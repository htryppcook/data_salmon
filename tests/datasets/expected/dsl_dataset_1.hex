0000000061626364000000016162636400000002616263640000000361626364000000046162636400000005616263640000000661626364000000076162636400000008616263640000000961626364