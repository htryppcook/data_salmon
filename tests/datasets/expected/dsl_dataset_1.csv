field0,field1
0,abcd
1,abcd
2,abcd
3,abcd
4,abcd
5,abcd
6,abcd
7,abcd
8,abcd
9,abcd
